README.md

```diff
@@ -43,6 +43,8 @@ Also, the [KDoc (similar to Javadoc, but for Kotlin)](https://floscher.gitlab.io
 * [Gubaer/josm-**scripting**-plugin](https://github.com/Gubaer/josm-scripting-plugin)
 * [JOSM/**pt_assistant**](https://gitlab.com/JOSM/plugin/pt_assistant)
 * [JOSM/**wikipedia**](https://gitlab.com/JOSM/plugin/wikipedia)
+* [Josm Review Plugin](https://github.com/DavidKarlas/JosmReviewPlugin)
+* [Josm Relation Validation](https://github.com/DavidKarlas/JosmRelationValidation)
```

Comment:
```suggestion
* [DavidKarlas/JosmRelationValidation (**Relation Validation Plugin**)](https://github.com/DavidKarlas/JosmRelationValidation)
* [DavidKarlas/JosmReviewPlugin (**Review Changes**)](https://github.com/DavidKarlas/JosmReviewPlugin)
* [Gubaer/josm-**scripting**-plugin](https://github.com/Gubaer/josm-scripting-plugin)
* [JOSM/**pt_assistant**](https://gitlab.com/JOSM/plugin/pt_assistant)
* [JOSM/**wikipedia**](https://gitlab.com/JOSM/plugin/wikipedia)
```

Thank you for the kind words, glad to hear you like the plugin! :tada: 

I modified the link text to match the other plugins in the list (path of the repository and the plugin name in bold).

You might consider renaming the plugins, so they don't use spaces, see https://josm.openstreetmap.de/wiki/DevelopersGuide/DevelopingPlugins#Namingtheplugin . It's probably not necessarily relevant for your plugins, because they are not managed by JOSM core developers, but it might still be possible that this could lead to problems. Something that matches `[a-zA-Z_-]+` would probably be best.